kuma/scrape/management/commands/scrape_document.py

```diff
@@ -33,19 +34,20 @@ def add_arguments(self, parser):
 
     def handle(self, *arg, **options):
         self.setup_logging(options['verbosity'])
-        host, ssl, path = self.parse_url_or_path(options['url'])
-        scraper = self.make_scraper(host=host, ssl=ssl)
-
-        params = {}
-        for param in ('force', 'translations', 'revisions', 'depth'):
-            if options[param]:
-                params[param] = options[param]
-        scraper.add_source("document", path, **params)
-
-        scraper.scrape()
-        source = scraper.sources['document:' + path]
-        if source.state == source.STATE_ERROR:
-            raise CommandError('Unable to scrape document "%s".' % path)
-
-        elif source.freshness == source.FRESH_NO and not options['force']:
-            self.stderr.write('Document "%s" already exists. Use --force to update.' % path)
+        for url in options['urls']:
+            host, ssl, path = self.parse_url_or_path(url)
+            scraper = self.make_scraper(host=host, ssl=ssl)
+
+            params = {}
+            for param in ('force', 'translations', 'revisions', 'depth'):
+                if options[param]:
+                    params[param] = options[param]
+            scraper.add_source("document", path, **params)
```

Comment:
Instead of running multiple scrapers, I would add multiple sources to the single scraper.

Reply:
The `self.make_scraper(...)` call depends on the URL. You *could* do something like:

```python
host, ssl, path = self.parse_url_or_path(urls[0])
scraper = self.make_scraper(host=host, ssl=ssl)
for url in options['urls']:
    host, ssl, path = self.parse_url_or_path(url)
    ...
    scraper.add_source("document", path, **params)
    ...
```

But that feels clunky and it's not a huge cost with creating the scraper since the use case is running this rarely on your laptop. It's rare that you run `scrape_document` and it's even more rare that you have more than 1 URLs.